tests/checkpoint/test_checkpoint.py

```diff
@@ -589,7 +590,12 @@ def test_checkpoint_run_dependencies_not_added_raises_error(
         ]
 
     @pytest.mark.unit
-    def test_checkpoint_run_no_actions(self, validation_definition: ValidationDefinition):
+    def test_checkpoint_run_no_actions(
+        self, validation_definition: ValidationDefinition, mocker: MockerFixture
+    ):
+        context = mocker.Mock(spec=AbstractDataContext)
+        set_context(project=context)
+        mocker.patch("great_expectations.checkpoint.checkpoint.submit_analytics_event")
```

Comment:
Do we have to mock this? Or can we disable usage stats such that we don't even care about this call? I'm pretty confident that CI won't send events like this

Reply:
thanks! i was just trying to do due diligence to not send events from tests, whether from CI or when triggered by a developer, but I can remove this is it isn't necessary.